lib/main_common.pm

```diff
@@ -832,7 +835,7 @@ sub load_inst_tests {
     if (get_var('ENCRYPT_CANCEL_EXISTING') || get_var('ENCRYPT_ACTIVATE_EXISTING')) {
         loadtest "installation/encrypted_volume_activation";
     }
-    if (get_var('MULTIPATH')) {
+    if (get_var('MULTIPATH') or defined(get_var('MULTIPATH_DIALOG_YES'))) {
```

Comment:
`defined` is not needed

Reply:
Yes, it is. Intention is to distinguish between the cases "unset" (get_var() returns undef) and "set to 0" (get_var returns "false") and have dedicated reactions to them.

As there are overall *three* cases to distinguish rather than the usual two, this handling of MULTIPATH_DIALOG_YES seemed to be the most natural way. An alternative like "1 means yes, 2 means no, false or undefined means 'no action'" looked too contrived IMHO. If there is a more desirable way I'd be glad to learn about it.

Reply:
This is quite confusing to rely on undefined variable or with content 0 or 1 as it quite differs from existing logic of test scheduling in main file. Undefined variable is usually treated as false, same result as it contains 0. What if you change variable name to `ISCSI_MULTIPATH_CONFIRM` with values `YES` /  `NO` and not defined?

Reply:
Point taken. Thanks for your proposal; I'll proceed to follow it. _Question:_ strictly spoken, this would entail the need to add a sanity check to multipath.pm against `ISCSI_MULTIPATH_CONFIRM` set to something else?

Reply:
Well, yes if not YES/yes/NO/no is not matched you can raise error. I was thinking further about variable name, but it don't have to be strictly tight ISCSI so it can be name just `MULTIPATH_CONFIRM`.

Reply:
Variable renamed in amended PR commit e150dcc (below). For the sake of simplicity every value which does not (sort of) match a "NO" (the "extraordinary" value) will be considered a "YES" .